src/routes/chat/+layout.svelte

```
@@ -2,19 +2,32 @@
 	import { Toaster } from '$lib/components/ui/sonner/index.js';
 	import * as Sidebar from '$lib/components/ui/sidebar/index.js';
 	import AppSidebar from '$lib/components/ui/app-sidebar/app-sidebar.svelte';
+	import { Separator } from '$lib/components/ui/separator';
+	import { chatStore, selectedChatStore } from '$lib/store/chat.store';
 
 	let { children } = $props();
+	let selectedChat = $derived<Chat | undefined>(
```

ESLint

[error] 'Chat' is not defined.

(no-undef)